docs/generate_docs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+cd $( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+OUTPUT_DIR='../dowhy-docs'
+
+PYTHONPATH=$(pwd)/.. sphinx-multiversion source ${OUTPUT_DIR}
+
+STABLE_VERSION=$(git describe --tags --abbrev=0 --match='v*')
+
+echo "<html>
```

Comment:
I did not understand why this is needed. Does it print the header for each html file?

Reply:
So, I'm not sure to which statement your comment refers to exactly. If this is about the `index.html` file created here, this is needed because the outputs of `sphinx-multiversion` go into sub-directories named after the release version or `master`. So effectively, there is no `index.html` at https://py-why.github.com/dowhy. This is what gets created here and all this HTML is doing, is, it immediately redirects to the latest release or `STABLE_VERSION` as determined by the `git describe` command.

You can see the rendered index.html here: https://github.com/petergtz/dowhy/blob/gh-pages/index.html#L1-L5

Reply:
understood, thanks